clinica/pipelines/t1_freesurfer_atlas/t1_freesurfer_atlas_utils.py

```diff
@@ -1,16 +1,96 @@
 # coding: utf8
 
 
+from pathlib import Path
+
 from clinica.utils.stream import cprint
 
 
+def path_cross(caps_directory: str, to_process_with_atlases: tuple, path: Path):
+    import os
+
+    hemisphere = Path(path.stem).stem
+    atlas_name = Path(path.stem).suffix[1:].split("_")[0]
+    sub, ses = to_process_with_atlases[1].split("_")
+    subject_dir = (
+        caps_directory
+        + "/subjects/"
+        + sub
+        + "/"
+        + ses
+        + "/t1/freesurfer_cross_sectional"
+    )
+
+    path_to_freesurfer_cross = subject_dir + "/" + to_process_with_atlases[1]
```

Comment:
you can use the pathlib's syntax of concatenation for cross platform support

Reply:
I have corrected a great number of these. Do tell me if you find some I have missed !

Reply:
Apparently this part of the code is not used. Please remove all dead code.